### PR TITLE
lscpumf: add z15 Model T02

### PR DESCRIPTION
**Pull request by sharkcz**

### CODE REVIEW

**Comment by sharkcz** [comment]
Machine type `8562` is handled in `get_counter()`, but missing in `machine_name()`, so be consistent.

**Comment by sharkcz** [comment]
ping :-)

File: cpumf/lscpumf.c [hunk 1]
```diff
@@ -2572,6 +2572,7 @@ static const char *machine_name(void)
 	case 3906:	return "IBM z14";
 	case 3907:	return "IBM z14 ZR1";
 	case 8561:	return "IBM z15";
+	case 8562:	return "IBM z15 Model T02";
```
Comment (hbrueckner) [inline cpumf/lscpumf.c:2575]: Hi Dan,

I just checked the current branding guidelines and it should be "IBM z15; Model T02" which is now a bit different compared to the z14 naming.

**Review by hbrueckner — COMMENTED** [review]
> ping :-)

pong :)

I just added a short remark on the naming. Thomas is currently out and I would ask him for a final review when he is back.

File: cpumf/lscpumf.c (hunk 1 above)
Comment (sharkcz) [reply cpumf/lscpumf.c]: thanks, PR updated, there might be other places/projects that would need an update of the branding

**Comment by tmricht** [comment]
@sharkcz Dan, can you please omit the semicolon after z15.
As hbrückner posted in his append.

**Comment by sharkcz** [comment]
done, I thought he wants me to add the semicolon :-)

**Pull request closed without merging**